### PR TITLE
KeywordBear: Propose patches for all todos

### PR DESCRIPTION
**Pull request by Techievena**
Propose patches to the user for all the keywords found
in the file.

Fixes https://github.com/coala/coala-bears/issues/1274

### CODE REVIEW

**Comment by aptrishu** [comment]
commit body -> Propose patches to remove all the comments in which keywords are found
in the file.

File: bears/general/KeywordBear.py [hunk 1]
```diff
@@ -95,6 +95,7 @@ def run(self,
             A regular expression to search for matching keywords in a file.
         '''
         comments = _get_comments(dependency_results)
+        comments = list(comments)
```
Comment (Mixih) [inline bears/general/KeywordBear.py]: why not `list(_get_comments(dependency_results))` ?

Comment (Techievena) [inline bears/general/KeywordBear.py]: Sorry I was in haste of working on new issues. Forgot it.:sweat_smile:

**Comment by aptrishu** [comment]
ack 619fcaa4dcc43d85f65969d75713de43eb76eddf

**Comment by aptrishu** [comment]
rebase needed.

**Comment by aptrishu** [comment]
@rultor merge

**Comment by aptrishu** [comment]
ack b8d3820726163bebe1d8200cd84480dbed843311

**Comment by aptrishu** [comment]
@rultor merge

**Pull request merged**